Source/Widgets/Timeline/Timeline.js

```diff
@@ -232,13 +232,54 @@ Timeline.prototype.destroy = function () {
 };
 
 /**
- * @private
+ * Shows an highlighted area on the timeline
+ *
+ * @param {Object} config
+ * @param {String|Color} config.color The highlight background color. Can be a css string or a Cesium.Color
+ * @param {number} config.height The height of the highlight area
+ * @param {number} config.base The bottom margin between the highlight range and the bottom of the timeline
+ * @param {JulianDate} config.startTime The start time of the highlight range
+ * @param {JulianDate} config.endTime The end time of the highlight range
+ *
+ * @returns {String} The new HighlightRange id
  */
-Timeline.prototype.addHighlightRange = function (color, heightInPx, base) {
-  var newHighlightRange = new TimelineHighlightRange(color, heightInPx, base);
+Timeline.prototype.addHighlightRange = function (config, heightInPx, base) {
+  // TODO::Deprecation part - remove on deprecation
+  if (typeof config === "string") {
+    var newDeprecatedHighlightRange = new TimelineHighlightRange(
+      config,
+      heightInPx,
+      base
+    );
+    this._highlightRanges.push(newDeprecatedHighlightRange);
+    this.resize();
+    return newDeprecatedHighlightRange;
```

Comment:
should this be `newDeprecatedHighlightRange.id`?

Reply:
@hpinkos this is the legacy code. It returns what a person using the old (private) method expects to receive (see the legacy test I've added to the timeline spec).